modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js

```diff
@@ -64,6 +64,7 @@ const Radio = ({
 			<div className="ddm__radio" onBlur={onBlur} onFocus={onFocus}>
 				<ClayRadioGroup
 					inline={inline}
+					name={name}
```

Comment:
Note to reviewer:

> A radio group is defined by giving each of radio buttons in the group the same [name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name). Once a radio group is established, selecting any radio button in that group automatically deselects any currently-selected radio button in the same group.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#value).

Also: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/radiogroup_role